registry-scanner/pkg/common/constants.go

```diff
@@ -56,6 +56,11 @@ const (
 	HelmPrefix                = "helmvalues"
 )
 
+// DefaultTargetFilePattern configurations related to the write-back functionality
+const DefaultTargetFilePattern = ".argocd-source-%s_%s.yaml"
+const DefaultTargetFilePatternWithoutNamespace = ".argocd-source-%s.yaml"
+const DefaultHelmValuesFilename = "values.yaml"
+
```

Comment:
I see both image-updater and registry-scanner have `constants.go`. Do we want to keep only fields required by registry-scanner in registry-scanner/constants.go, and remove those not used by registry-scanner?

Reply:
Yes, I am working on a logic to separate them out. It needs a bit of refactoring. I will have the PR updated.

Reply:
I will move this change to a different PR.